src/components/Camera/style.css

```diff
@@ -1,3 +1,36 @@
+@import (less) "../Theme/constants.css";
+
+.rectangle {
+  //default value of react-webcam https://github.com/cezary/react-webcam
+  @width-frame: 640;
+  @height-frame: 480;
+
+  //Spec of overlay
+  @width-percent: 80%;
+  @aspect-ratio: 1.57;
+
+  /*
+    From the equation below one can find out @height-percent
+    aspect-ratio = (width-frame * width-percent) / (height-frame * height-percent)
```

Comment:
Okay, so all of this styling stuff is just moved code

But this comment seems to pretty much just mirror the code, right? No harm in removing :)